src/main.js

```diff
@@ -47,27 +47,29 @@ function createWindow() {
   });
 
   // set up some chrome extensions
-  const {
-    default: installExtension,
-    REACT_DEVELOPER_TOOLS,
-    REACT_PERF,
-  } = require('electron-devtools-installer');
-
-  require('electron-debug')({ showDevTools: true });
-
-  const ChromeLens = {
-    // ID of the extension (https://chrome.google.com/webstore/detail/chromelens/idikgljglpfilbhaboonnpnnincjhjkd)
-    id: 'idikgljglpfilbhaboonnpnnincjhjkd',
-    electron: '>=1.2.1',
-  };
-
-  const extensions = [REACT_DEVELOPER_TOOLS, REACT_PERF, ChromeLens];
-
-  for (const extension of extensions) {
-    try {
-      installExtension(extension);
-    } catch (e) {
-      console.error(chalk.red(`[ELECTRON] Extension installation failed`), e);
+  if (process.env.NODE_ENV !== 'development') {
+    const {
+      default: installExtension,
+      REACT_DEVELOPER_TOOLS,
+      REACT_PERF,
+    } = require('electron-devtools-installer');
+
+    require('electron-debug')({ showDevTools: true });
+
+    const ChromeLens = {
+      // ID of the extension (https://chrome.google.com/webstore/detail/chromelens/idikgljglpfilbhaboonnpnnincjhjkd)
+      id: 'idikgljglpfilbhaboonnpnnincjhjkd',
+      electron: '>=1.2.1',
+    };
+
+    const extensions = [REACT_DEVELOPER_TOOLS, REACT_PERF, ChromeLens];
+
+    for (const extension of extensions) {
+      try {
+        installExtension(extension);
+      } catch (e) {
+        console.error(chalk.red(`[ELECTRON] Extension installation failed`), e);
+      }
```

Comment:
It turns out, weirdly, that this was partially the cause of the GPU weirdness; sending info to devtools o_O

I did initially like the idea of devtools being enabled for production, so that folks could poke around under the hood with Guppy... but that's a pretty fringe benefit, and not worth it if it means performance suffers

Reply:
Weird that it impacts the GPU indeed